tests/test_exacttarget_bookmarks.py

```diff
@@ -0,0 +1,122 @@
+import datetime
+from base import ExactTargetBase
+import tap_tester.connections as connections
+import tap_tester.menagerie as menagerie
+import tap_tester.runner as runner
+import datetime
+import dateutil.parser
+import pytz
+
+class ExactTargetBookmarks(ExactTargetBase):
+    def name(self):
+        return "tap_tester_exacttarget_bookmarks"
+
+    def convert_state_to_utc(self, date_str):
+        date_object = dateutil.parser.parse(date_str)
+        date_object_utc = date_object.astimezone(tz=pytz.UTC)
+        return datetime.datetime.strftime(date_object_utc, "%Y-%m-%dT%H:%M:%SZ")
+
+    def test_run(self):
+        conn_id_1 = connections.ensure_connection(self)
+        runner.run_check_mode(self, conn_id_1)
+
+        expected_streams = self.streams_to_select()
+
+        found_catalogs_1 = menagerie.get_catalogs(conn_id_1)
+        self.select_found_catalogs(conn_id_1, found_catalogs_1, only_streams=expected_streams)
+
+        # Run a sync job using orchestrator
+        first_sync_record_count = self.run_and_verify_sync(conn_id_1)
+        first_sync_records = runner.get_records_from_target_output()
+        first_sync_bookmarks = menagerie.get_state(conn_id_1)
+
+        ##########################################################################
+        ### Update Start Date
+        ##########################################################################
+
+        self.START_DATE = '2021-01-01T00:00:00Z'
```

Comment:
For the Bookmark test why we are updating start_date? We need to update the state.

Reply:
Updated the code change to use state.